CHANGELOG.md

```diff
@@ -8,6 +8,12 @@ The format is based on [Keep a Changelog][], and this project adheres to
 
 ## [Unreleased][]
 
+### Changed
+
+-  Buildpacks are no longer installed to `$HOME/.yourbase/tools`. They are now
+   installed into the per-build context cache. You can safely remove
+   `$HOME/.yourbase/tools` to reclaim disk space.
+
```

Comment:
Any reason we can't remove this for them?

Reply:
I'm always a little nervous about tools that delete without user interaction (what if you still have an old version of yb hanging around? etc.). We could check for the directory on startup and log a message?

Reply:
(I'm going to merge this to unblock some follow-on PRs I had queued, but I'm happy to come back and address this if you want it included.)

Reply:
If you think it's generally going to be < 1 GB then it's probably not worth the prompt. If not, I think it is worth it to be a good neighbor rather than silently eating space for potentially years.

Reply:
Opened ch-2788